tests/unit/r2i/onnxrt_openvino/parameters.cc

```diff
@@ -18,287 +18,40 @@
-  r2i::RuntimeError error;
-  r2i::onnxrt_openvino::Parameters parameters;
-
-  std::shared_ptr<r2i::IEngine> engine(new r2i::onnxrt_openvino::Engine);
-  std::shared_ptr<r2i::IModel> model(new r2i::onnxrt::Model);
-
-  error = parameters.Configure(engine, model);
-
-  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode ());
-
-  std::shared_ptr<r2i::IEngine> internalEngine = parameters.GetEngine();
-
-  POINTERS_EQUAL(internalEngine.get(), engine.get());
-}
-
-TEST (OnnxrtOpenVinoParameters, SetUndefinedIntegerParameter) {
-  r2i::RuntimeError error;
-  r2i::onnxrt_openvino::Parameters parameters;
-  int value;
-
-  std::shared_ptr<r2i::IEngine> engine(new r2i::onnxrt_openvino::Engine);
-  std::shared_ptr<r2i::IModel> model(new r2i::onnxrt::Model);
-
-  error = parameters.Configure(engine, model);
-  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode ());
-
-  value = 0;
-  error = parameters.Set("undefined-parameter", value);
-  LONGS_EQUAL (r2i::RuntimeError::Code::INVALID_FRAMEWORK_PARAMETER,
-               error.GetCode ());
-}
-
-TEST (OnnxrtOpenVinoParameters, GetUndefinedIntegerParameter) {
-  r2i::RuntimeError error;
-  r2i::onnxrt_openvino::Parameters parameters;
-  int value;
-
-  std::shared_ptr<r2i::IEngine> engine(new r2i::onnxrt_openvino::Engine);
-  std::shared_ptr<r2i::IModel> model(new r2i::onnxrt::Model);
-
-  error = parameters.Configure(engine, model);
-  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode ());
-
-  error = parameters.Get("undefined-parameter", value);
-  LONGS_EQUAL (r2i::RuntimeError::Code::INVALID_FRAMEWORK_PARAMETER,
-               error.GetCode ());
-}
-
-TEST (OnnxrtOpenVinoParameters, SetUndefinedStringParameter) {
-  r2i::RuntimeError error;
-  r2i::onnxrt_openvino::Parameters parameters;
-  std::string value;
-
-  std::shared_ptr<r2i::IEngine> engine(new r2i::onnxrt_openvino::Engine);
-  std::shared_ptr<r2i::IModel> model(new r2i::onnxrt::Model);
-
-  error = parameters.Configure(engine, model);
-  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode ());
-
-  value = "test";
-  error = parameters.Set("undefined-parameter", value);
-  LONGS_EQUAL (r2i::RuntimeError::Code::INVALID_FRAMEWORK_PARAMETER,
-               error.GetCode ());
-}
-
-TEST (OnnxrtOpenVinoParameters, GetUndefinedStringParameter) {
-  r2i::RuntimeError error;
-  r2i::onnxrt_openvino::Parameters parameters;
-  std::string value;
-
-  std::shared_ptr<r2i::IEngine> engine(new r2i::onnxrt_openvino::Engine);
-  std::shared_ptr<r2i::IModel> model(new r2i::onnxrt::Model);
-
-  error = parameters.Configure(engine, model);
-  LONGS_EQUAL (r2i::RuntimeError::Code::EOK, error.GetCode ());
-
-  error = parameters.Get("undefined-parameter", value);
-  LONGS_EQUAL (r2i::RuntimeError::Code::INVALID_FRAMEWORK_PARAMETER,
-               error.GetCode ());
-}
-
-TEST (OnnxrtOpenVinoParameters, SetAndGetLogId) {
+TEST (OnnxrtOpenVinoParameters, SetHardwareIdAtWrongEngineState) {
```

Comment:
Sees like we are missing a bunch of tests. Wrong values, wrong types, unknown parameters, etc...

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/481b2ed3b0e12d38d57cd863cd25238d368d85f0)